src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -895,7 +906,20 @@ def main(argv):
     arguments.append(constants.VALUE_PROFILE_ARGUMENT)
     fuzzing_strategies.append(strategy.VALUE_PROFILE_STRATEGY)
 
-  if do_fork():
+  use_dataflow_tracing = False
+
+  # Depends on the presense of DFSan instrumented build.
+  dataflow_build_path = environment.get_value('DATAFLOW_BUILD_DIR_???')
```

Comment:
???

Reply:
Just a sign that it's a sketch :) Will use a normal name or some other way to pass the build dir.